src/pyflex/config.py

```diff
@@ -24,7 +24,8 @@ def __init__(self, min_period, max_period, stalta_waterlevel=0.07,
                  tshift_acceptance_level=10.0, tshift_reference=0.0,
                  dlna_acceptance_level=1.3, dlna_reference=0.0,
                  cc_acceptance_level=0.7, s2n_limit=1.5, earth_model="ak135",
-                 min_surface_wave_velocity=3.0,
+                 min_surface_wave_velocity=3.5,
```

Comment:
please don't change existing default values. There are already people using this and we don't want their results to change just because they update pyflex.